bin/rules.d/openmoteb.rules

```diff
@@ -0,0 +1,2 @@
+# open nodes openmoteb
+SUBSYSTEM=="tty", SUBSYSTEMS=="usb", ATTRS{idVendor}=="0403" , ATTRS{idProduct}=="6010" , ENV{ID_USB_INTERFACE_NUM}=="01", MODE="0664", GROUP="dialout", SYMLINK+="iotlab/ttyON_OPENMOTEB"
```

Comment:
I think I understand why the product and vendor id are the same as Fox and nearly the same as M3 and A8-M3 open nodes/control node.
To access the serial port your setup uses an FTDI USB to serial converter. These IDs are FTDI IDs: https://www.ftdichip.com/Support/Documents/TechnicalNotes/TN_100_USB_VID-PID_Guidelines.pdf

That might be worth mentioning it here or somewhere in the open node documentation (in the class doc string for example) or in both places.